UPGRADING.rst

```diff
@@ -11,6 +11,37 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#4673 Eliminate burner accounts
+===============================
+
+When this PR lands in the dev deployment, in `platform-hca-dev` Google cloud
```

Comment:
`platform-hca-dev` project was the only one containing burner accounts. Should Google cloud projects be included either way?